src/groundStation/commandParser.py

```diff
@@ -118,8 +118,11 @@ def parseReturnValue(self, src, dst, dport, data, length):
                 return outputObj
 
             else:
-                outputObj[retVal] = np.frombuffer(
-                    data, dtype=returns[retVal], count=1, offset=idx)[0]
+                try:
+                    outputObj[retVal] = np.frombuffer(
+                        data, dtype=returns[retVal], count=1, offset=idx)[0]
+                except:
+                    return outputObj
```

Comment:
```suggestion
                    outputObj[retVal] = None
```
I guess this sets each retVal to None for ones where numpy errors out